dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/Tracer.kt

```diff
@@ -22,6 +27,29 @@ import java.util.Properties
  */
 class Tracer internal constructor(config: Config, writer: TraceWriter) : DDTracer(config, writer) {
 
+    private val random: Random = SecureRandom()
```

Comment:
shouldn't this be added from the Builder ?

Reply:
Indeed 😕 So why are the test green ?

Reply:
> So why are the test green ?

Because there's another
```diff
+        private var random: Random = SecureRandom()
```
in the `Builder` 🤔.